test-it/karma/publicapi/FPSync.Test.js

```diff
@@ -0,0 +1,67 @@
+describe('The Vireo CoreHelpers setFPSyncFunction api', function () {
+    'use strict';
+    // Reference aliases
+    var Vireo = window.NationalInstruments.Vireo.Vireo;
+    var vireoRunner = window.testHelpers.vireoRunner;
+    var fixtures = window.testHelpers.fixtures;
+
+    var vireo;
+
+    var publicApiFPSyncSimpleViaUrl = fixtures.convertToAbsoluteFromFixturesDir('publicapi/FPSyncSimple.via');
+    var publicApiFPSyncUtf8ViaUrl = fixtures.convertToAbsoluteFromFixturesDir('publicapi/FPSyncUtf8.via');
+
+    beforeAll(function (done) {
+        fixtures.preloadAbsoluteUrls([
+            publicApiFPSyncSimpleViaUrl,
+            publicApiFPSyncUtf8ViaUrl
+        ], done);
+    });
+
+    beforeEach(function () {
+        vireo = new Vireo();
+    });
+
+    it('can perform a simple fpsync', function (done) {
+        var tracker = jasmine.createSpy();
+        vireo.coreHelpers.setFPSyncFunction(tracker);
+        var runSlicesAsync = vireoRunner.rebootAndLoadVia(vireo, publicApiFPSyncSimpleViaUrl);
+
+        runSlicesAsync(function (rawPrint, rawPrintError) {
+            expect(rawPrint).toBeEmptyString();
+            expect(rawPrintError).toBeEmptyString();
+            expect(tracker).toHaveBeenCalled();
+            expect(tracker).toHaveBeenCalledWith('Hello World');
+
+            done();
+        });
+    });
+
+    it('can perform a fpsync with UTF8 characters', function (done) {
+        var viName = 'MyVI';
+
+        var trackerCalls = [];
+        var tracker = function (fpSyncString) {
+            var myDouble = vireo.eggShell.readDouble(viName, 'myDouble');
+            trackerCalls.push({
+                myDouble: myDouble,
+                fpSyncString: fpSyncString
+            });
+        };
+
+        vireo.coreHelpers.setFPSyncFunction(tracker);
+        var runSlicesAsync = vireoRunner.rebootAndLoadVia(vireo, publicApiFPSyncUtf8ViaUrl);
+
+        runSlicesAsync(function (rawPrint, rawPrintError) {
+            var fpSyncString = 'Iñtërnâtiônàlizætiøn\0☃💩';
```

Comment:
We could but I don't think it would be clearer.

The code point for 💩 is \x1F4A9

The JS string representation is "UTF-16" so each "code unit" is 16-bit. So that 💩 code point is broken into two code points ([with the surrogate pair algorithm](https://en.wikipedia.org/wiki/UTF-16#U.2B10000_to_U.2B10FFFF)) which is two "code units" or 4 bytes total hex:
\xD83D \x56489

The JS length of the string is two.

The Vireo string representation is "UTF-8" so each "code unit" is 8-bit. So that 💩 code point is converted into 4 "code units"  or 4 bytes hex: 
\xF0\x9F\x92\xA9

The Vireo length of the string is four.

So placing the hex would be more precise in the test but I don't think it would be clearer at demonstrating what is being tested.